backends/dpdk/dpdkProgram.cpp

```diff
@@ -508,7 +508,10 @@ bool ConvertToDpdkControl::preorder(const IR::P4Action *a) {
     for (auto i : helper->get_instr())
         stmt_list->push_back(i);
 
-    auto action = new IR::DpdkAction(*stmt_list, a->name, *a->parameters);
+    auto actName = a->name.name;
+    if (a->name.toString() == "NoAction")
```

Comment:
toString() or name?

Reply:
Here it is toString() only. 
 .name would have the name with **_declid** suffix. like NoAction_1,NoAction_2 etc